hazelcast/src/main/java/com/hazelcast/instance/impl/ClusterTopologyIntent.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.instance.impl;
+
+public enum ClusterTopologyIntent {
```

Comment:
can you please add concise javaDoc about what is intent and why we need it?

Reply:
added in 4f840b6f9e7a8b6f775e7ee9951358689ce5d022